.github/workflows/falco.yml

```diff
@@ -0,0 +1,25 @@
+name: Get and Update Latest Releases
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version tag for falco'
+        default: 'latest'
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        description: 'version tag for falco'
+        default: 'latest'
+        required: true
+        type: string
+
+jobs:
+  TBD:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check the triggered call
+        run: |
+          echo "Version: ${{ inputs.image_tag }}"
```

Comment:
What is this workflow for?

We will need to trigger the deploy workflow being added in #105 passing the 3 inputs.

Since we're using the REST API to fetch the releases I think we should also use it to trigger.

Reply:
actually my purpose for the version as the only input here is as we are sending the version which we want to install rest of the checks which all component are present in the projects/project.json might be possible from the child job to take out and use it

still if you want to add more info we can.

and the updatedProjects are stored in /tmp/updatedProjects.json and it is used to set the environment variable for particular proj so that we can add a if statement for each job and it will be triggered only if its true